### PR TITLE
Feature: invitations/users autorefresh

### PR DESCRIPTION
**Pull request by NicolasRichel**
https://platform-dev-invitations-autorefresh.bimdata.io

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] My code follows the code style of this project.
- [x] I have tested my code.
- [x] I want to run the tests for the commits of this PR

### CODE REVIEW

File: src/views/space-board/SpaceBoard.vue [hunk 1]
```diff
@@ -130,10 +130,18 @@ export default {
       project => project.name
     );
 
+    useInterval(
+      async () => await Promise.all([
+        loadSpaceUsers(currentSpace.value),
```
Comment (Amoki) [inline src/views/space-board/SpaceBoard.vue]: Is there a reason to reload clouds?

Comment (NicolasRichel) [reply src/views/space-board/SpaceBoard.vue]: This method will reload cloud **users**.

File: src/views/invitations/Invitations.vue [hunk 2]
```diff
@@ -234,23 +236,25 @@ export default {
...
         isLoadingAllInvit.value = true;
-        await acceptInvitations(invitationListPending.value);
+        await acceptInvitations(pendingInvitations.value);
         isLoadingAllInvit.value = false;
       }
     };
 
     const onDenyInvitation = async invitation => {
-      await denyInvitation(invitation);
+      await declineInvitation(invitation);
```
Comment (Kurtil) [inline src/views/invitations/Invitations.vue]: useless async await

File: src/views/invitations/Invitations.vue [hunk 3]
```diff
@@ -234,23 +236,25 @@ export default {
...
     };
 
+    useInterval(async () => await loadUserInvitations(), 5000);
```
Comment (Kurtil) [inline src/views/invitations/Invitations.vue]: useless async await

File: src/views/project-board/ProjectBoard.vue [hunk 4]
```diff
@@ -140,24 +147,26 @@ export default {
...
-    provide(
-      "shouldSubscribe",
-      computed(
-        () =>
-          currentSpace.value.isFree &&
-          currentSpace.value.isUserOrga &&
-          isFullTotal(spaceSubInfo.value)
-      )
+    useInterval(
+      async () => {
```
Comment (Kurtil) [inline src/views/project-board/ProjectBoard.vue]: useInterval only handles synchronous callbacks... async await not really useful here.

**Review by Kurtil — CHANGES_REQUESTED** [review]

**Review by Kurtil — APPROVED** [review]

**Review by GabBimdata — APPROVED** [review]

**Pull request merged**